<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>login</class>
 <widget class="QWidget" name="login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>543</width>
    <height>543</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="passwordLabel">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>420</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>密码：</string>
   </property>
  </widget>
  <widget class="QLabel" name="accountLabel">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>360</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>帐号:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="logonBtn">
   <property name="geometry">
    <rect>
     <x>183</x>
     <y>183</y>
     <width>121</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="signupBtn">
   <property name="geometry">
    <rect>
     <x>183</x>
     <y>267</y>
     <width>121</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="passwordText">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>450</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="accountText">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>390</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="accountLabel_2">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>230</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>登录</string>
   </property>
  </widget>
  <widget class="QLabel" name="accountLabel_3">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>290</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>注册</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
